### PR TITLE
Yunsi/gp 85/api to get user info

### PR DESCRIPTION
**Pull request by yun-si**
## Changes

1. Three API to get young people info（including function, route, testing function）
    - people feature
    - prefer furniture
    - prefer place
 2. Edit insert data of posgresql
 
## What you need to do
1. click http://localhost:7877/pg_init/ to reinsert data

### Optional Testing
click http://localhost:7877/get_preference_furniture/1
You will get return below.
```
{
  "message": [
    {
      "Preference_ID": 1,
      "Preference_House_Furniture": "衣櫃"
    },
    {
      "Preference_ID": 1,
      "Preference_House_Furniture": "床"
    }
  ]
}
```

### CODE REVIEW

File: tests/unit/app/test_route.py [hunk 1]
```diff
@@ -68,3 +68,23 @@ def test_mongodb_init_status_code():
 def test_mongodb_init_test_status_code():
     response = client.get("/mongo_init_test/")
     assert response.status_code == 200
+
+
+def test_get_young_info():
+    response = client.get("/get_young_info/")
+    assert response.status_code == 200
```
Comment (frafishen) [inline tests/unit/app/test_route.py:75]: [nit] It would be better if the code comments were in English.

**Review by frafishen — APPROVED** [review]
LGTM!!!

**Comment by frafishen** [comment]
Is it ok to fix the readme in this PR?
https://github.com/Fang-4-Group/Graduation-Project#setup-and-start
image

File: tests/unit/app/test_route.py (hunk 1 above)
Comment (yun-si) [reply tests/unit/app/test_route.py:75]: change it

**Pull request merged**